rust/operator-binary/src/airflow_controller.rs

```diff
@@ -60,6 +63,14 @@ pub struct Ctx {
     pub product_config: ProductConfigManager,
 }
 
+lazy_static! {
+    pub static ref WEBSERVER_SECRET_KEY: String = rand::thread_rng()
+        .sample_iter(&Alphanumeric)
+        .take(15)
+        .map(char::from)
+        .collect();
+}
+
```

Comment:
Also WEBSERVER_SECRET_KEY should not change over the restart of Airflow instcances or operators as this will invalidate user sessions (additional reason to do this)

Reply:
As discussed: have replace the unused SECRET_KEY with AIRFLOW__WEBSERVER__SECRET_KEY